src/MessageTrait.php

```diff
@@ -342,6 +350,31 @@ private function setHeaders(array $originalHeaders): void
         $this->headers     = $headers;
     }
 
+    /**
+     * @todo Replace this method with a call to ! empty($array) &&
+     *     array_is_list once minimum PHP version is 8.1.
+     */
+    private function arrayIsList(array $array): bool
+    {
+        if (empty($array)) {
+            return false;
+        }
+
+        if (function_exists('array_is_list')) {
```

Comment:
can't we just drop PHP <8.1? 
Did that in laminas-servicemanager for v4 as well, already seen other components dropping 8.0 so I guess that would fit?